package.json

```diff
@@ -64,6 +60,8 @@
     "lodash.defaults": "3.1.1",
     "react": "0.13.1",
     "react-hammerjs": "0.2.2",
+    "react-imageloader": "1.2.0",
```

Comment:
let's also add a short section in the README.md here along the lines of "dependencies for common tasks" so people don't reinvent the wheel all the time, and have a quick place to find out what we use for which purpose